src/Acuminator/Acuminator.Utilities/Properties/AssemblyInfo.cs

```diff
@@ -1,30 +0,0 @@
-﻿using System.Resources;
```

Comment:
What happened with AssemblyInfo file?

Reply:
in net standard it is not required - it is embedded in csproj
https://montemagno.com/how-to-convert-a-pcl-library-to-net-standard-and-keep-git-history/